tests/v2/validation/provisioning/k3s/custom_cluster_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build (provisioning.k3s && p0) || validation || (provisioning.k3s && !release && !release.oob && !short)
```

Comment:
Close but this is how it works 

- k3s,p0 just runs k3s custom cluster tests
- validation will run absolutely everything so if validation is present this test will build.
- and also if provisioning.k3s is provided, this will run all k3s provisioning tests, but we don't want to run them if the other tags are provided. Make sense? 

Yeah but we use the regex for our other testing, so updating the test name doesn't make sense imo. I think if we're not happy with tag naming we can always update it. Naming test names is a bigger push, since we already use the regex.

Reply:
I think it's a great idea to standardize test names in this process as well, however i.e. for the upgrade HA case, we had to partially standardize the provisioning test names beforehand - especially the subtests. I agree, I think it's a bigger push across the tests and can be a separate future task, I feel like scoping the test runs with tags and regexes gives us a bit more flexibility without discouraging contributions and focuses more on the recurring runs end